### PR TITLE
Documentation updates for Audience Manager

### PR DESCRIPTION
**Pull request by addb**

### CODE REVIEW

File: using-mobile-extensions/adobe-audience-manager/README.md [hunk 1]
```diff
@@ -22,15 +22,37 @@ To get started with **Audience Manager**, follow these steps:
...
+#### Objective-C
+  ```objectivec
+  #import "ACPCore.h"
+  #import "ACPAudience.h"
+  #import "ACPIdentity.h"
+  ```
+
+#### Swift
+  ```swift
+   import "ACPCore"
```
Comment (nporter-adbe) [inline using-mobile-extensions/adobe-audience-manager/README.md]: These should be just without the quotes
```
import ACPCore
import ACPAudience
```

When installing via Cocoapods the module map only creates ACPCore which has an umbrella header that auto imports ACPIdentity. https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/pull/31

File: using-mobile-extensions/adobe-audience-manager/README.md [hunk 2]
```diff
@@ -22,15 +22,36 @@ To get started with **Audience Manager**, follow these steps:
...
+  #import "ACPIdentity.h"
+  ```
+
+#### Swift
+  ```swift
+   import ACPCore
+   import ACPAudience
+  ```
+
+   **Important**: Audience Manager depends on the Identity extension and is automatically included in the Core pod. When installing manually, ensure that you have also added the `ACPIdentity.a` to your project.
```
Comment (nporter-adbe) [inline using-mobile-extensions/adobe-audience-manager/README.md]: I think we should replace `ACPIdentity.a` with `libACPIdentity_iOS.a`.
https://github.com/Adobe-Marketing-Cloud/acp-sdks/blob/master/iOS/ACPCore/libACPIdentity_iOS.a

File: using-mobile-extensions/adobe-audience-manager/README.md [hunk 3]
```diff
@@ -31,6 +31,8 @@ To get started with **Audience Manager**, follow these steps:
 
 #### Java
 `import com.adobe.marketing.mobile.*;`
+ **Important**: Audience Manager depends on the Identity extension and is automatically included in the Core pod. When installing the Audience Manager Extension manually, ensure that you have added the `identity-1.x.x.aar` file to your project.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-audience-manager/README.md]: - In "When installing the Audience Manager Extension manually", please change "Extension" to "extension".

- In "ensure that you have added the `identity-1.x.x.aar` file to your project.", please remove "have".

File: using-mobile-extensions/adobe-audience-manager/README.md [hunk 4]
```diff
@@ -49,27 +51,11 @@ To get started with **Audience Manager**, follow these steps:
    import ACPAudience
   ```
 
-   **Important**: Audience Manager depends on the Identity extension and is automatically included in the Core pod. When installing manually, ensure that you have also added the `ACPIdentity.a` to your project.
+   **Important**: Audience Manager depends on the Identity extension and is automatically included in the Core pod. When installing the Audience Manager Extension manually, ensure that you have also added the `libACPIdentity_iOS.a` to your project.
```
Comment (rekrishn) [inline using-mobile-extensions/adobe-audience-manager/README.md]: - In "When installing the Audience Manager Extension manually", please change "Extension" to "extension".

- In "ensure that you have added the `identity-1.x.x.aar` file to your project.", please remove "have".

**Pull request merged**